src/main/java/uk/gov/ons/census/fwmt/rmadapter/service/impl/MessageListenerService.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.ons.census.fwmt.rmadapter.service.impl;
+
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
```

Comment:
For this type of task I think we dont need it in the ervice layer, service layer is more for business functionality. I think its safe to move the simpleMessageListenerContainer.start(); and stop to the controller class and delete this class. But well remembered